tensorboard/webapp/runs/views/runs_table/runs_group_menu_button_component.ts

```diff
@@ -24,18 +32,58 @@ import {ChangeDetectionStrategy, Component} from '@angular/core';
     >
       <mat-icon svgIcon="palette_24px"></mat-icon>
     </button>
-    <mat-menu #groupByMenu="matMenu">
-      <button mat-menu-item>
-        <span>Experiments</span>
+    <mat-menu #groupByMenu="matMenu" class="run-table-color-group-by">
+      <div class="label">Color runs by</div>
+      <button
+        mat-menu-item
+        role="menuitemradio"
+        (click)="onGroupByChange.emit({key: GroupByKey.EXPERIMENT})"
+      >
+        <span>
+          <mat-icon
+            *ngIf="selectedGroupBy.key === GroupByKey.EXPERIMENT"
+            svgIcon="done_24px"
+          ></mat-icon>
+        </span>
+        <label>Experiments</label>
```

Comment:
Just to clarify. Do we want `s` at the end or not?
`Experiment` or `Experiments`
(In the doc we used singular not plural to communicate, but I don't object to change it to plural)
I don't know which makes more sense but just want to be consistent so let me know your decision and I'll follow.